src/repositories/reviewRepo.js

```diff
@@ -5,6 +5,12 @@ const save = (review) => {
     return newReview.save();
 };
 
+const deleteReview = async (reviewId) => {
+    const review = await Review.findById(reviewId);
+    review.deleteOne();
+};
```

Comment:
Why are we not returning anything from a function, we should avoid creating a void function because.
1. I case of void we dont is going on inside function .
2. Creates problem while writing  proper Unit test.
3. either return the deleted review or boolean.

Reply:
Also we can return a promise here, no need to resolve, because that will adds to redundant await in the other layers. We can await  the promise at controller layer